docs/cloud/index.md

```diff
@@ -0,0 +1,18 @@
+# DAOS in the Cloud
+
+In addition to on-prem deployments, DAOS is available as part of
+cloud offerings.
+
+## DAOS in Google Cloud
+
+DAOS is available as an HPC storage option in Google Cloud.
+ 
+To learn more about DAOS in Google Cloud, please refer to the
+[Google Cloud DAOS repository](https://github.com/daos-stack/google-cloud-daos)
+on github and the 
+[Google Cloud Platform documentation for DAOS](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/community/modules/file-system/Intel-DAOS/README.md).
+
+A discussion forum for technical questions and support
+for DAOS in Google Cloud is available in the
+[DAOS on Google Cloud](https://daos-stack.slack.com/archives/C03GLTLHA59)
+Slack channel.
```

Comment:
I am not sure to understand the difference between this index.md file and docs/QSG/cloud.md. It looks to me that we are duplicating the content ... I probably missed something.

Reply:
one is the high-level description, the other one is focusing on the actual deployment steps. took this from ivan's original request and added some verbiage around the hyperlinks, maybe this caused it to look too similar (but the links are going to different GCP documents...)